charts/ndb-operator/Chart.yaml

```diff
@@ -22,8 +22,14 @@ maintainers:
 icon: https://www.nutanix.com/content/dam/nutanix/global/icons/products/svg/Nutanix-Era-40.svg
 annotations:
   artifacthub.io/changes: |
+    - kind: fix
+      description: "Upgraded CONTROLLER_TOOLS_VERSION from v0.9.2 to v0.14.0"
+    - kind: fix
+      description: "Upgraded go version in go.mod from 1.20 to 1.21.7"
+    - kind: fix
+      description: "Removed samples folder reference which was located in config/manifests/kustomization"
     - kind: security
-      description: "Bumped kube-rbac-proxy to v0.15.0"
+      description: "Upgraded kube-rbac-proxy version from v0.15.0 to v0.16.0 and other go modules to newer versions"
```

Comment:
Nit: This ArtifactHub Helm chart need not provide information about go modules since this does not contain anything related to go. All of these are internal details and are more suited for the ndb-operator github releases.
Same with samples folder and controller tools version. 

WDYT?

Reply:
Sounds good doing this instead
```
    - kind: fix
      description: "Upgraded internal dependencies."
    - kind: security
      description: "Upgraded kube-rbac-proxy from v0.15.0 to v0.17.0."
```